spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MetricsProperties.java

```diff
@@ -133,6 +133,20 @@ public void setMaxUriTags(int maxUriTags) {
 			 */
 			private boolean autoTimeRequests = true;
 
+			/**
+			 * Default percentiles when "autoTimeRequests=true" and @Timed annotation is
```

Comment:
I don't like the idea of referring one property from another property. The canonical format is not camelCase anyway.